wiki/Tournaments/RMoT_Invitational/5/en.md

```diff
@@ -0,0 +1,168 @@
+- ![][flag_GB] [Olii](/users/7683305 "Olii")
+- ![][flag_BR] [PlayPen](/users/5094351 "PlayPen")
+- ![][flag_FR] [Prosdokies](/users/3852436 "Prosdokies")
+- ![][flag_RU] [Well Halen](/users/8660244 "Well Halen")
+- ![][flag_UA] [Xelite](/users/7793117 "Xelite")
+
+*Notice: Klein- and Nogad gave up their places in the Invitational, and iTzAshraf and \_Suzuha reached semifinals twice. Due to this, 4 people who have not reached semifinals were selected to participate in the tournament: filler players.*
+
+The filler players are chosen according to a few criteria in the following priority:
+
+1. Having participated in and reached QF (quarterfinals) once out of the last 4 RMoT's
+2. Number of times reached QF
+3. Number of times participated in RMoT
+4. Current pp
+
+![][flag_MY] [Aasheda\_](/users/7614055 "Aasheda\_"), ![][flag_US] [Likean00b](/users/4860447 "Likean00b"), ![][flag_PL] [My Angel Erwin](/users/6814521 "My Angel Erwin") and ![][flag_RU] [Skif4a8929](/users/6700953 "Skif4a8929") gained the right to participate in the Invitational, ![][flag_RU] [Skif4a8929](/users/6700953 "Skif4a8929") being seeded lower than the other three.
+
+ ## Links
+ 
+- [Discussion thread](/community/forums/topics/741126)
+- **[Challonge](https://challonge.com/rmotinvitational5)**
+
+------------------------------------
+
+## Mappools
+
+There is only one mappool for all matches, made of beatmaps from the last four RMoT mappools. There are six Tiebreaker beatmaps, one for each stage. Every Tiebreaker from the last four RMoT's and two new beatmaps are picked for Tiebreaker.
+
+**[Download the mappack here! (182 MB)](http://www.mediafire.com/file/2telo31u7xasasn/RMoT%20Invitational%20%235%20-%20Mappool.zip)**
+
+- NoMod
+  - [BTS - FIRE (Nathan) [apple's Insane]](/beatmaps/1068523)
+  - [Falcom Sound Team jdk - Over Drive (Kite) [Insane]](/beatmaps/127762)
+  - [Memme - Dajiahao (Erhu ver.) (Kamio Misuzu) [Extra]](/beatmaps/643565)
+  - [Omoi - Snow Drive (01.23) (Kroytz) [pk's Insane]](/beatmaps/1083055)
+  - [Suzumu - Kakumeisei ousama densenbyou (tutuhaha) [Frey's Insane]](/beatmaps/688305)
+  - [ZUN - Hartmann's Youkai Girl (Gomo Psivarh) [Ipvarsh]](/beatmaps/84964)
+- Hidden
+  - [Draw the Emotional - Morning glow (UnitedWeSin) [Lunatic]](/beatmaps/363298)
+  - [Nightwish - Dead to the World (Luvdic) [Insane]](/beatmaps/363298)
+- HardRock
+  - [44teru-k - Yami ~ Shoujo no Yume , Kyomu no Sekai ~ ([Twinkle]) [Hyper]](/beatmaps/374403)
+  - [SawanoHiroyuki[nZk]:mica - Summer Tears (appleeaterx) [Insane]](/beatmaps/892298)
+- DoubleTime
+  - [LiSA - traumerei (Knotts) [Karia's Hard]](/beatmaps/483885)
+  - [Nishino Kana - Day 7 (yanlu) [Collab]](/beatmaps/251475)
+- FreeMod
+  - [HO-KAGO TEA TIME - Singing! (CDFA) [Insane]](/beatmaps/164603)
+  - [syrela - Pulsation (Lasse) [Insane]](/beatmaps/1436970)
+  - [xi - Akasha (Atsuro) [FCL's Hyper]](/beatmaps/909551)
+- Tiebreaker
+  1. [u's - Moment Ring (Chaoslitz) [wkyik's Insane]](/beatmaps/1030059)
```

Comment:
Any reason to use a numbered list here? also those need to be bolded

Reply:
`There are six Tiebreaker beatmaps, one for each stage.` i guess that's the reason